tests/loggen/loggen_helper.c

```diff
@@ -148,7 +148,15 @@ int connect_unix_domain_socket(int sock_type, const char *path)
 
   DEBUG("unix domain socket: %s\n",path);
   saun.sun_family = AF_UNIX;
-  strncpy(saun.sun_path, path, sizeof(saun.sun_path));
+
+  gsize max_target_path_size = sizeof(saun.sun_path);
+  if (strlen(path) >= max_target_path_size)
```

Comment:
Just a few thoughts out-load, consider it as a note (this is not a change request):
This is not equivalent in a sense that previous code handled not null terminated `path`, and this is not.
Anyhow neither version handles the fact that the `sun_path` shall be null terminated :)

Reply:
That's true, thanks for pointing out :) Relying on the fact that `path` is coming from `argv[1]` is not the best idea, so a forced NULL-termination would be a good idea.

Reply:
Sorry, I was wrong. The NULL-termination was never handled, `strncpy` won't terminate a string if `src` is not terminated.

Reply:
We agreed that the current state of this PR is good enough, it is assumed that `path` is a string (null-terminated).